tests/multi_wlan/01_ap_sta.py

```diff
@@ -0,0 +1,117 @@
+# Basic Wi-Fi MAC layer test where one device creates an Access Point and the
+# other device connects to it as a Station. Also tests channel assignment (where
+# possible) and disconnection.
+
+try:
+    import network
+
+    network.WLAN
+except (ImportError, NameError):
+    print("SKIP")
+    raise SystemExit
+
+import os
+import sys
+import time
+
+CHANNEL = 8
+
+# Note that on slower Wi-Fi stacks this bumps up against the run-multitests.py
+# timeout which expects <10s between lines of output. We work around this by
+# logging something half way through the wait_for loop...
+CONNECT_TIMEOUT = 15000
```

Comment:
I've hit this 10s limit a few times.  Happy to just increase it to 15s to make this test simpler, and for future tests.

Alternatively (but probably out of scope for this PR) could add a special debug output function on the `multitest` class and just ignore that data, eg:
```py
multitest.output_debug("random data that shouldn't affect test result")
```
(See related `output_metric`.)

Reply:
Let's leave this as-is.  In the future we can support `unittest` in the multitests to improve things.